### PR TITLE
add direct CSI docs

### PR DESCRIPTION
**Pull request by harshavardhana**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ MinIO-Operator brings native MinIO, [MCS](https://github.com/minio/mcs), and [KE
 
 - Kubernetes version v1.17.0 and above for compatibility. MinIO Operator uses `k8s/client-go` v0.18.0.
 - `kubectl` configured to refer to a Kubernetes cluster.
-- Create the required PVs as [explained here](https://github.com/minio/minio-operator/blob/master/docs/creating-pv-manually.md).
+- Create the required PVs using [direct CSI driver](https://github.com/minio/minio-operator/blob/master/docs/using-direct-csi.md).
```
Comment (nitisht) [inline README.md:23]: Actually we can have two options here e.g. 

```
Create the required PVs using
- Direct CSI Driver
- Manual PV Creation
- Auto provisioning of local PVs
```

Comment (harshavardhana) [reply README.md:23]: No I think only one way @nitisht all other methods are not recommended.

File: docs/using-direct-csi.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+## Install Direct CSI Driver
+
+### Set the environment variables
+
+```sh
+cat << EOF > default.env
+DIRECT_CSI_DRIVER_PATHS=/var/lib/direct-csi-driver/data{1...4}
+DIRECT_CSI_DRIVER_COMMON_CONTAINER_ROOT=/var/lib/direct-csi-driver
+DIRECT_CSI_DRIVER_COMMON_HOST_ROOT=/var/lib/direct-csi-driver
```
Comment (kerneltime) [inline docs/using-direct-csi.md:11]: What are these environment variables and where are they used?

File: docs/using-direct-csi.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
+# Using Direct CSI Driver [![Slack](https://slack.min.io/slack?type=svg)](https://slack.min.io) [![Docker Pulls](https://img.shields.io/docker/pulls/minio/k8s-operator.svg?maxAge=604800)](https://hub.docker.com/r/minio/k8s-operator)
+
+## Install Direct CSI Driver
+
```
Comment (kerneltime) [inline docs/using-direct-csi.md:4]: The buttons mostly will be ignored above, we should include the link here.

File: docs/using-direct-csi.md (hunk 2 above)
Comment (harshavardhana) [reply docs/using-direct-csi.md:11]: It's for kustomize @kerneltime https://github.com/minio/direct-csi-driver#steps-to-run

**Pull request merged**